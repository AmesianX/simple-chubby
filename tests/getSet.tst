# test get and set APIs
fileOpen /test 8
# return fdId: 1
get 1
set 1 hello
get 1

fileDelete 1
fileClose 1

exit