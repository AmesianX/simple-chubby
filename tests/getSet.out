1

Succeeded
hello
Deleted
